src/java/io/bazel/rulesscala/coverage/instrumenter/JacocoInstrumenter.java

```diff
@@ -0,0 +1,105 @@
+package io.bazel.rulesscala.coverage.instrumenter;
+
+import io.bazel.rulesscala.jar.JarCreator;
+import io.bazel.rulesscala.worker.GenericWorker;
+import io.bazel.rulesscala.worker.Processor;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.FileVisitor;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import org.jacoco.core.instr.Instrumenter;
+import org.jacoco.core.runtime.OfflineInstrumentationAccessGenerator;
+
+public final class JacocoInstrumenter implements Processor {
+
+    public static void main(String[] args) throws Exception {
+        (new Worker()).run(args);
+    }
+
+    private static final class Worker extends GenericWorker {
+        public Worker() {
+            super(new JacocoInstrumenter());
+        }
+    }
+
+    @Override
+    public void processRequest(List < String > args) {
```

Comment:
weird spacing on the `List<String>` here.

Reply:
I can fix it. Not sure how it wound up this way (perhaps my editor's Java formatter got carried away?).